spec/test_cbw_api.py

```diff
@@ -108,3 +108,37 @@ def test_remote_accesses():
             assert isinstance(response, list) is True
             for remote in response:
                 assert isinstance(remote, CBWRemoteAccess) is True
+
+    @staticmethod
+    def test_create_remote_access():
+        """Tests for method remote_accesses"""
+        client = CBWApi(API_URL, API_KEY, SECRET_KEY)
+
+        info = {"type": "CbwRam::RemoteAccess::Ssh::WithPassword",
+                "address": "X.X.X.X",
+                "port": "22",
+                "login": "loginssh",
+                "password": "passwordssh",
+                "key": "",  # precises the key of the connection
+                "node": "master"  # precises the Cyberwatch source of the connection,
+                }
+
+        with vcr.use_cassette('spec/fixtures/vcr_cassettes/create_remote_access.yaml'):
+            response = client.create_remote_access(info)
+
+            assert response is True
+
+        info = {"type": "CbwRam::RemoteAccess::Ssh::WithPassword",
```

Comment:
if the address is the only field changing, you can just write `info["address"] = ""`